### DISCUSSION TITLE
Surface Keyboard Compatability

### DISCUSSION THREAD

**Discussion by 1234cj** [linux-surface/linux-surface#1456] [General]
Is the stand alone keyboard you can buy from Microsoft - the "Surface Keyboard" - compatible with this project?

**Comment by qzed** [comment]
As far as I understand, that should be a standard Bluetooth keyboard. So in theory it should work out-of-the-box on any Linux system and there shouldn't be any patches required (but I never had one and haven't heard any reports, so I can't guarantee this).

**Reply by 1234cj** [reply]
Hi, 

Thanks for the reply. The reason I ask is it uses the SAM chip and there are some reports of it not working properly. No worries if you don't know, I'll probably go with something else.

**Reply by qzed** [reply]
The SAM chip is basically a serial device. So it's hard-wired and I'm pretty sure it doesn't use that (or at least doesn't expose that via bluetooth). I can't say whether it uses some custom bluetooth/HID extensions though, which could cause the reports you mentioned.
